packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -79,7 +79,8 @@ const ruleset: any = {
     // RPC Code: RPC-Async-V1-02
     //PUT
     ProvisioningStateSpecifiedForLROPut: {
-      description: 'A LRO PUT operations response schema must have "ProvisioningState" property specified.',
```

Comment:
each rule should have a separate doc file which is named with the rule name (kebab case) ,could you add the new rule file to the /docs

Reply:
The pipeline will generate the rule link according to the rule name, so if lacks the corresponding rule doc file, the link will be invalid.

Reply:
@jianyexi  Its a same rule, implemented separately for PUT, PATCH & DELETE
would this still need diff docs?

Reply:
yes, as they're different rules in implementation now.

Reply:
ok so looks like this will cause a broken link?